### ISSUE TITLE
Declarations not included

### ISSUE THREAD

**Issue by amcdnl** [rollup/rollup-plugin-typescript#54]
It would be nice if the `declaration: true` flag would be honored in the output.

**Comment by thgreasi** [comment]
:+1: I currently have to use typescript on its own to generate es6 files and `.d.ts` files and then use rollup for bundling and generating a UMD dist.

**Comment by thgreasi** [comment]
[rollup-plugin-typescript2](https://github.com/ezolenko/rollup-plugin-typescript2/blob/master/README.md#declarations) says that it supports it. I will give it a try but I would prefer to use the official rollup plugin for typescript.

**Comment by ncphillips** [comment]
I've had to switch to `rollup-plugin-typescript2` as well. Honestly, I don't see how this plugin is useful in for creating public packages without this feature.

**Comment by janbiasi** [comment]
Still no update on this? I have to sadly agree to @ncphillips ...

**Comment by mkay581** [comment]
Yup, I've moved to `rollup-plugin-typescript2` as well. Any idea on a timeline of when this will be implemented?

**Comment by onbjerg** [comment]
Guys, still nothing?

**Comment by couellet** [comment]
Had to switch to `rollup-plugin-typescript2` too, and it's 10 times slower. Hope this is going to be fixed soon!

**Comment by bodograumann** [comment]
I use `tsc --emitDeclarationOnly` in addition to rollup. It‘s not that bad, but it‘s also not perfect.

**Comment by shellscape** [comment]
There are quite a few people commenting on support for this, but we don't have any open PRs to address this. We're _totally_ open to adding this, but we need you fine folks to help us. We're getting ready to move this plugin to a new home at https://github.com/rollup/plugins, and we have to do some spring cleaning of the issues to make that happen. We're going to close this one if no one from userland steps up to make the request happen. After that point, we'd encourage anyone still facing the issue to pitch in on the new repo and get this improvement in for the plugin.
